runtime/onert/backend/acl_cl/KernelGenerator.cc

```diff
@@ -703,19 +703,23 @@ void KernelGenerator::visit(const ir::operation::Transpose &node)
 {
   const auto ofm_idx{node.getOutputs().at(0)};
   const auto ifm_idx{node.getInputs().at(ir::operation::Transpose::Input::INPUT)};
-  const auto &perm{node.param().perm};
+  const auto perm_idx{node.getInputs() at(ir::operation::Transpose::Input::PERM)};
```

Comment:
"." is missed out between "getInputs()" and "at()"

Reply:
Thanks, fixed the error.